avm/res/relay/namespace/main.json

```diff
@@ -447,7 +447,7 @@
         }
       ],
       "metadata": {
-        "description": "Optional. Authorization Rules for the Relay namespace."
+        "description": "Optional. Authorization Rules for the Relay Namespace."
```

Comment:
Shouldn't this update the templateHash?

Reply:
Interesting, but it didn't. And the pipeline finishes green...

Reply:
I just re-ran the Set-AVMModule, no changes....

Reply:
@eriqua it doesn't because it's just a casing change. Just tested via `bicep build` myself.